### PR TITLE
app-forensics/rkhunter: patch https://bugs.gentoo.org/645454 and change maintainer

### PR DESCRIPTION
**Pull request by JordyZomer**
Hey,

Bug: https://bugs.gentoo.org/645454
Closes: https://bugs.gentoo.org/645454

It creates a temporary dir for rkhunter so it can run without triggering the error "Invalid TMPDIR configuration option: Non-existent pathname: /var/lib/rkhunter/tmp".

I would also like to maintain this package, I added myself to the metadata file with commit d4a0cfa .

Kind Regards,

Jordy Zomer

### CODE REVIEW

File: app-forensics/rkhunter/rkhunter-1.4.4.ebuild [hunk 1]
```diff
@@ -52,6 +52,8 @@ src_install() {
 	newexe "${FILESDIR}/${PN}-1.3.cron" ${PN}
 
 	newbashcomp "${FILESDIR}/${PN}.bash-completion" ${PN}
+
+	dodir /var/lib/${PN}/tmp
```
Comment (titanofold) [inline app-forensics/rkhunter/rkhunter-1.4.4.ebuild]: Use `keepdir` instead. Empty directories will be cleaned automatically.

File: app-forensics/rkhunter/metadata.xml [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
+    <email>admin@jordyzomer.nl</email>
+    <name>Jordy Zomer</name>
+  </maintainer>
   <upstream>
     <remote-id type="sourceforge">rkhunter</remote-id>
   </upstream>
+  <maintainer type="project">
+    <email>proxy-maint@gentoo.org</email>
+    <name>Proxy Maintainers</name>
+  </maintainer>
```
Comment (titanofold) [inline app-forensics/rkhunter/metadata.xml]: Keep maintainers together.

**Comment by JordyZomer** [comment]
@titanofold  Is it okay now? :)

File: app-forensics/rkhunter/Manifest [hunk 3]
```diff
@@ -1,2 +1,8 @@
...
+DIST rkhunter-1.4.4.tar.gz 297626 SHA256 a8807c83f9f325312df05aa215fa75ad697c7a16163175363c2066baa26dda77 SHA512 87e9c617220765678cc4519eee27d1d56185c3a7fb1d6338c8fb984ac4f5176c31bb54b69e1de615d66a0cf1e72b672e66b368e37851a459def69463cbb8661e
+EBUILD rkhunter-1.4.2.ebuild 1411 SHA256 d02e361bcd6e16ec86e351a307156b7c779bb8d684129791beeeef5771b77492 SHA512 e4b2f70f6417f3a44d918ab310460dd45f7fae760458f7b9804752a79beffabea832dd195c85efeb8e31d1b71d7196b1df70a8cdc5a562d0a4ef575888820ada
+EBUILD rkhunter-1.4.4.ebuild 1462 SHA256 cfa28389e0fd0153b79b35d5bcbf7d6ae004a6328e71b9e41d2f64ee6a11bbd1 SHA512 8e0de63eff6d0933000d22a207ea3db1e161db6c864853c5dfe7201d2a406a67b6cfcf3342fc6e55cf39a7ae077421d6bfce01407c8228c3cb4e9fc5e7e87009
+MISC metadata.xml 250 SHA256 b7958445feebc53822c26943827589ae43981f57398e4907fd2f2b176f34b17b SHA512 ab31238a72b35a1f33056cf1736628bbf367fc20f4dc93735d11617ed668f295567e1865844d727e666de07c56d84f8b5cfd1eec41c91cd836789a420ea2dda9
```
Comment (titanofold) [inline app-forensics/rkhunter/Manifest]: Oh no!

Looks like you had something change for the metadata/layout.conf because this is going backwards.

**Comment by mgorny** [comment]
Ping.

**Comment by mgorny** [comment]
Closing due to inactivity. Feel free to reopen when you're back around.

**Pull request closed without merging**